cmd/surf/services/parallelfilesprocessor.go

```diff
@@ -82,3 +91,10 @@ func (p *ParallelFilesProcessor) Run(ctx context.Context,
 
 	return nil
 }
+
+func min(x, y int) int {
```

Comment:
❓ https://golang.org/pkg/math/#Min

Reply:
Heh. go's static typing means `x` and `y` need to be cast to `float64` before being used in `math.Min`. I just moved the `min` fn from another package (it was in the wrong place before), and used it to avoid the cast.